docs/source/whatsnew/1.0.0b5.rst

```diff
@@ -11,6 +11,14 @@ API Changes
 * :py:func:`solarforecastarbiter.reports.figures.output_svg` now takes a
   :py:class:`selenium.webdriver.remote.webdriver.WebDriver` as an optional
   argument (:pull:`345`)
+* Add metrics for deterministic event forecasts
+  :py:mod:`solarforecastarbiter.metrics.event.probability_of_detection`,
+  :py:mod:`solarforecastarbiter.metrics.event.false_alarm_ratio`,
+  :py:mod:`solarforecastarbiter.metrics.event.probability_of_false_detection`,
+  :py:mod:`solarforecastarbiter.metrics.event.critical_success_index`,
+  :py:mod:`solarforecastarbiter.metrics.event.event_bias`, and
+  :py:mod:`solarforecastarbiter.metrics.event.event_accuracy`. (:issue:`347`) (:pull:`348`)
```

Comment:
I think all of these ``:py:mod:`` should be ``:py:func:`` https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects

Reply:
It looks like previous `whatsnew` files used `:py:mod:`, but I agree it's better as `:py:func:`.